docs-src/content/using-cubic-sdk/_index.md

```diff
@@ -0,0 +1,13 @@
++++
+title = "Using Cubic-SDK"
+description = ""
+weight = 2
++++
+
+This section discusses the various aspects of using the SDK as a client.
+
+* [Installing the SDK](./installation)
+* [Setup A Connection](./connecting)
+* [Synchronous Recognition](./recognize)
+* [Streaming Recognition](./streaming)
```

Comment:
A lot of users get confused about which we actually recommend. We should somehow make it obvious that streaming is how we want users to use this API in most cases. Not sure how to best do it though.

Reply:
I've added a notice at the top of the Synchrounous page.  Hopefully that helps.